contracts/feeManager/FeeManager.sol

```diff
@@ -229,4 +232,25 @@ contract FeeManager is IFeeManager, Ownable {
 
         return funds;
     }
+
+    /**
+     * @notice Overrides the transferOwnership function of Ownable
+     * to remove FINANCIAL_OWNER role from previous _feesOwner
+     * @param newOwner The address of the new owner
+     */
+    // slither-disable-next-line missing-zero-check
+    function transferOwnership(address newOwner) public override {
```

Comment:
We should check, besides the zero check, that they new and old owner are the same and exit, to avoid transaction costs if no transfer is needed.

Reply:
Verification for previous owner added, it will revert with message `NewOwnerIsCurrentOwner()`